include/nuttx/lib/math.h

```diff
@@ -81,11 +81,17 @@
 #define INFINITY_F  (1.0F/0.0F)
 #define NAN_F       (0.0F/0.0F)
 
+#define INFINITY_L  (1.0F/0.0L)
+#define NAN_L       (0.0F/0.0L)
+
```

Comment:
Are these correct or are they supposed to be 1.0L/0.0L and 0.0L/0.0L? (i.e., is the F suffix correct on the two numerators?)

Reply:
Yes, you are right.

Reply:
I will fix. When I squash should it be `git rebase -i 94cf99f310409f8c86681a0ee7177f2bbfeaa24a` which seems to mean I will squash my just squashed commit and this new one? Are you sure it's not easier from github?

Reply:
> I will fix. When I squash should it be `git rebase -i 94cf99f310409f8c86681a0ee7177f2bbfeaa24a` which seems to mean I will squash my just squashed commit and this new one?

Yes, git rebase -i work correctly.

> Are you sure it's not easier from github?

If we use github merge, it will generate the commit message automatically by accumulating individual commit message which may not good as expect.